ibis/backends/base/sql/alchemy/datatypes.py

```diff
@@ -38,8 +38,9 @@ def table_from_schema(name, meta, schema, database: Optional[str] = None):
     dt.String: sa.Text,
     dt.Decimal: sa.NUMERIC,
     # Mantissa-based
-    dt.Float: sa.REAL,
-    dt.Double: sa.FLOAT,
+    dt.Float16: sa.REAL,
+    dt.Float32: sa.REAL,
+    dt.Float64: sa.REAL,
```

Comment:
`dt.Float64` maps to `sa.FLOAT`. `sa.REAL` is 32-bit floating point. Yeah, it's confusing, and taken from SQL 😑